charts/lagoon-remote/Chart.yaml

```diff
@@ -19,7 +19,7 @@ type: application
 # This is the chart version. This version number should be incremented each
 # time you make changes to the chart and its templates, including the app
 # version.
-version: 0.70.0
```

Comment:
I think this should be 0.71.0, as its a new feature - not a bugfix.

Reply:
ok - fixed